gobblin-service/src/test/java/org/apache/gobblin/service/modules/flow/MultiHopFlowCompilerTest.java

```diff
@@ -640,7 +640,18 @@ public void testCompileCombinedDatasetFlow() throws Exception {
     Assert.assertTrue(jobConfig2.getString(ConfigurableGlobDatasetFinder.DATASET_FINDER_PATTERN_KEY).endsWith("{dataset0,dataset1,dataset2}"));
   }
 
-  @Test (dependsOnMethods = "testCompileMultiDatasetFlow")
+  @Test (dependsOnMethods = "testCompileCombinedDatasetFlow")
+  public void testUnresovedFlow() throws Exception {
+    FlowSpec spec = createFlowSpec("flow/flow5.conf", "HDFS-1", "HDFS-3", false, false);
+
+    Dag<JobExecutionPlan> dag = specCompiler.compileFlow(spec);
+
+    Assert.assertTrue(dag.isEmpty());
+    Assert.assertEquals(spec.getCompilationErrors().size(), 1);
+    Assert.assertTrue(spec.getCompilationErrors().iterator().next().contains("user.to.proxy"));
+  }
+
+  @Test (dependsOnMethods = "testUnresovedFlow")
```

Comment:
does it depend on testUnresovedFlow ?

Reply:
Just following the pattern for the rest of the test, I think the reason is that the setup only needs to be done once then.